<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>audio_sounds_document</class>
 <widget class="QWidget" name="audio_sounds_document">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Audio - Sounds</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="sounds_label">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Sounds</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <layout class="QGridLayout" name="sound_settings">
     <item row="0" column="0">
      <widget class="QLabel" name="tracks_label">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Tracks</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QGroupBox" name="unknown_box">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="title">
        <string>Unknown Variables</string>
       </property>
       <layout class="QFormLayout" name="formLayout">
        <property name="fieldGrowthPolicy">
         <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="unknown1_label">
          <property name="text">
           <string>Unknown 1:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="unknown1_edit">
          <property name="font">
           <font>
            <family>Monospace</family>
           </font>
          </property>
          <property name="maxLength">
           <number>11</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="unknown2_label">
          <property name="text">
           <string>Unknown 2:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="unknown2_edit">
          <property name="font">
           <font>
            <family>Monospace</family>
           </font>
          </property>
          <property name="maxLength">
           <number>11</number>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="unknown3_label">
          <property name="text">
           <string>Unknown 3:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="unknown3_edit">
          <property name="font">
           <font>
            <family>Monospace</family>
           </font>
          </property>
          <property name="maxLength">
           <number>11</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QGroupBox" name="properties_box">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="title">
        <string>Properties</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QFormLayout" name="category_box">
          <item row="0" column="0">
           <widget class="QLabel" name="category_label">
            <property name="text">
             <string>Category:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="category_list">
            <property name="currentIndex">
             <number>0</number>
            </property>
            <item>
             <property name="text">
              <string>Default</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Music</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>ImpactsAndDeath</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>UI</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Narration</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>UniqueNarration</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>PORTAL VO</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="reverb_label">
            <property name="text">
             <string>Reverb:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <layout class="QVBoxLayout" name="reverb_list">
            <item>
             <widget class="QRadioButton" name="reverb_none">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>None</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="reverb_microsoft">
              <property name="text">
               <string>Microsoft Reverb</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QCheckBox" name="property_6">
          <property name="text">
           <string>Vocalization Attenuation</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="property_2">
          <property name="text">
           <string>Narration Attenuation</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="property_1">
          <property name="text">
           <string>Distance Attenuation</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="property_7">
          <property name="text">
           <string>Music Attenuation</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="property_5">
          <property name="text">
           <string>Reverb Distance Attenuation</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="property_4">
          <property name="text">
           <string>Menu Low Pass Frequency</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="property_3">
          <property name="text">
           <string>Game Low Pass Frequency</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="1">
      <layout class="QVBoxLayout" name="tracks_box">
       <item>
        <widget class="QLabel" name="wave_bank_files_label">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Wave Bank Files</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="wave_bank_files_list">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="contextMenuPolicy">
          <enum>Qt::ActionsContextMenu</enum>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QListWidget" name="tracks_list">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::ActionsContextMenu</enum>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QListWidget" name="sounds_list">
     <property name="cursor" stdset="0">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::ActionsContextMenu</enum>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
